lib/exec.js

```diff
@@ -45,28 +48,46 @@ async function exec (cmd, args = [], opts = {}) {
         reject(new Error(`Standard input '${err.syscall}' error: ${err.stack}`));
       });
     }
+    let stdoutBufferSize = 0;
     if (proc.stdout) {
       proc.stdout.on('error', (err) => {
         reject(new Error(`Standard output '${err.syscall}' error: ${err.stack}`));
       });
       // keep track of stdout if we have not said not to
-      if (!opts.ignoreOutput) {
+      if (opts.ignoreOutput) {
+        // https://github.com/nodejs/node/issues/4236
+        proc.stdout.on('data', () => {});
```

Comment:
rather than listening to the event with a noop, could we also just ... not listen to the event?

Reply:
have you checked the bug description above?

Reply:
nope, makes sense now!